inc/Engine/Optimization/RUCSS/Database/Tables/UsedCSS.php

```diff
@@ -42,6 +42,7 @@ class UsedCSS extends Table {
 		20220920 => 'make_status_column_index_instead_queue_name',
 		20221104 => 'add_error_columns',
 		20231010 => 'add_submitted_at_column',
+		20231031 => 'add_not_proceed_before_column',
```

Comment:
Plz change the $version above to make the DB changes happen

Reply:
I've done it, I always forget it ...